### ISSUE TITLE
TF-slim models label_offset is not handled

### ISSUE THREAD

**Issue by mschrimpf** [brain-score/candidate_models#16]
all the models that have a non-zero label offset (i.e. TensorFlow slim models with a background class at index zero) are outputting the wrong ImageNet logits. E.g. `inception_v1` thereby produces a score of 0 on the ImageNet benchmark.
